tools/coqmktop.ml

```diff
@@ -272,7 +272,7 @@ let main () =
     (* - We add topstart.cmo explicitly because we shunted ocamlmktop wrapper.
        - With the coq .cma, we MUST use the -linkall option. *)
     let args =
-      "-linkall" :: "-rectypes" :: "-w" :: "-31" :: flags @ copts @ options @
+      "-linkall" :: "-rectypes" :: "-safe-string" :: "-w" :: "-31" :: flags @ copts @ options @
```

Comment:
Indeed it seems that we could just take this from `Coq_config` in the style of #540